src/feature/playerhistory/playerhistory.controller.ts

```diff
@@ -0,0 +1,86 @@
+import {
+    ClientConnectedEvent,
+    ClientDisconnectedEvent,
+    ClientStateChangedEvent,
+    collection,
+    RpcEvent,
+  } from '@earnkeeper/ekp-sdk';
+  import {
+    AbstractController,
+    ApmService,
+    ClientService,
+    logger,
+  } from '@earnkeeper/ekp-sdk-nestjs';
+  import { Injectable } from '@nestjs/common';
+import { PlayerHistoryForm } from 'src/util/forms/player-history-form';
+  import { DEFAULT_HISTORY_FORM, DEFAULT_LEADERBOARD_FORM} from '../../util';
+  import { PlayerhistoryService } from './playerhistory.service';
+  import { PlayerhistoryDocument } from './ui/playerhistory.document';
+  import Playerhistory from './ui/playerhistory.uielement';
+  
+  const COLLECTION_NAME = collection(PlayerhistoryDocument);
+  const PATH = 'history';
+  
+  @Injectable()
+  export class PlayerHistoryController extends AbstractController {
+    constructor(
+      clientService: ClientService,
+      private PlayerhistoryService: PlayerhistoryService,
```

Comment:
Pay attention to your case for your variables and classes. Classes should use UpperCamelCase. Variables and properties should use lowerCamelCase. 

Google has a good style guide for typescript here: https://google.github.io/styleguide/tsguide.html#identifiers